feature-libs/cart/import-export/core/config/import-export-config.ts

```diff
@@ -10,5 +11,8 @@ export abstract class ImportExportConfig {
   importExport: {
     fileValidity?: FileValidity;
     file: { separator: string };
+    export: {
```

Comment:
will be additional columns required or optional?

Reply:
I changed config to optional for export